docs/examples/notebooks/03-Integrator.ipynb

```diff
@@ -0,0 +1,212 @@
+    "    fpga_ens = FpgaPesEnsembleNetwork(\n",
+    "        board,  # The board to use (from above)\n",
+    "        n_neurons=100,  # The number of neurons to use in the ensemble\n",
+    "        dimensions=1,  # 1 dimensions\n",
+    "        learning_rate=0,  # No learning for this example\n",
+    "        recurrence=1  # Activate the recurrent connection\n",
+    "    )\n",
+    "\n",
+    "    # Setting `recurrence=1` creates a `recurrent` connection object\n",
+    "    # with the identity transform. You can set the attributes on this\n",
+    "    # recurrent connection object, such as `.function` and `.transform`\n",
+    "    fpga_ens.recurrent.synapse = tau  # `nengo.LowPass(tau)`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 5: Connect Everything Together\n",
+    "The recurrent connection is housed on the FPGA device and is created as part of the `FpgaPesEnsembleNetwork` object, so we actually only need to connect the stimulus to the enesemble now!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with model:\n",
+    "    # Connect the stimulus\n",
+    "    nengo.Connection(input_node, fpga_ens.input,\n",
+    "                     synapse=tau, transform=tau)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 6: Add Probes to Collect Data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with model:\n",
+    "    # The original input\n",
+    "    stim_p = nengo.Probe(input_node)\n",
+    "\n",
+    "    # The output from the FPGA ensemble\n",
+    "    # (filtered with a 10ms post-synaptic filter)\n",
+    "    output_p = nengo.Probe(fpga_ens.output, synapse=0.01)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 7: Run the Model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with nengo_fpga.Simulator(model) as sim:\n",
+    "    sim.run(6)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 8: Plot the Results:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure(figsize=(8, 5))\n",
+    "plt.plot(sim.trange(), sim.data[stim_p], 'k--', label=\"Input\")\n",
+    "plt.plot(sim.trange(), sim.data[output_p], label=\"Integrator output\")\n",
+    "plt.legend();"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The plot above shows the neurons effectively integrating the input signal, albeit with a little drift. Try playing around with the number of neurons in the FPGA ensemble as well as the synaptic time constant (`tau`) to see how it effects performance!"
```

Comment:
Should also include the blurb from the nengo integrator notebook (https://www.nengo.ai/nengo/examples/dynamics/integrator.html) about why the drift is there.